layers/+lang/c-c++/config.el

```diff
@@ -47,3 +50,33 @@
 
 (defvar c-c++-default-mode-for-headers 'c-mode
   "Default mode to open header files. Can be `c-mode' or `c++-mode'.")
+
+;; c-c++-lsp-backend variables
+(defvar c-c++-lsp-cache-dir nil
+  "Cache directory. Absolute and relative paths supported.")
+
+(defvar c-c++-lsp-executable nil
+  "Path to cquery/ccls executable (default value assumes it's in the path)")
+
+(defvar c-c++-lsp-project-whitelist nil
+  "A list of project directory patterns for which lsp-c-c++ should be
+initialized. This overrides `c-c++-lsp-project-blacklist'.")
+
+(defvar c-c++-lsp-project-blacklist nil
+  "A list of project root patterns for which lsp-c-c++ shouldn't be
+initialized. `c-c++-lsp-project-whitelist' is checked first, then this,
+if no pattern matches the project root, lsp-c-c++ will be initialized.")
+
+(defvar c-c++-lsp-sem-highlight-method 'font-lock
+  "Set to 'font-lock or 'overlay to enable semantic highlighting")
+
+(defvar c-c++-lsp-sem-highlight-rainbow nil
+  "When non-nil, use rainbow semantic highlighting")
+
+;; I've left cquery/ccls -extra-init-params separate for now, as one has defaults while the other doesn't
+;; Just to facilitate switching between the two easily
+(defvar c-c++-lsp-extra-init-params '(:cacheFormat "msgpack")
```

Comment:
Technically ccls does not use "msgpack" but I've just made change to ccls. So as long as the initial is `"m"`, the compact binary serialization format will be used for both ccls and cquery

Reply:
Great - thanks